File: apps/cli/src/lib/env.ts
export function getVSTProject(): string | undefined {
  return process.env.VST_PROJECT;
}

export function getVSTWorktree(): string | undefined {
  return process.env.VST_WORKTREE;
}

export function getVSTSession(): string | undefined {
  return process.env.VST_SESSION;
}

export function getVSTDaemonUrl(): string | undefined {
  return process.env.VST_DAEMON_URL;
}

File: rust/vst-cli/src/env.rs
use std::env;

/// Read `VST_PROJECT` environment variable.
pub fn get_vst_project() -> Option<String> {
    env::var("VST_PROJECT").ok()
}

/// Read `VST_WORKTREE` environment variable.
pub fn get_vst_worktree() -> Option<String> {
    env::var("VST_WORKTREE").ok()
}

/// Read `VST_SESSION` environment variable.
pub fn get_vst_session() -> Option<String> {
    env::var("VST_SESSION").ok()
}

/// Read `VST_DAEMON_URL` environment variable.
pub fn get_vst_daemon_url() -> Option<String> {
    env::var("VST_DAEMON_URL").ok()
}
